CRM/Batch/BAO/Batch.php

```diff
@@ -648,10 +648,11 @@ public static function getBatchFinancialItems($entityID, $returnValues, $notPres
     }
 
     $from = "civicrm_financial_trxn
-LEFT JOIN civicrm_entity_financial_trxn ON civicrm_entity_financial_trxn.financial_trxn_id = civicrm_financial_trxn.id
+INNER JOIN civicrm_entity_financial_trxn ON civicrm_entity_financial_trxn.financial_trxn_id = civicrm_financial_trxn.id
+INNER JOIN civicrm_contribution ON (civicrm_contribution.id = civicrm_entity_financial_trxn.entity_id
```

Comment:
OK - I've worked through the clean up & it all makes sense, and is just clean-up as opposed to functionality change, as you say, except for the removal of

AND civicrm_entity_batch.entity_table = 'civicrm_financial_trxn'

which I'm still not clear about the reason for